Core/Core/Database/CoreData/Models/Tab.swift

```diff
@@ -46,6 +46,16 @@ public class Tab: NSManagedObject {
         get { return TabVisibility(rawValue: visibilityRaw) ?? .none }
         set { visibilityRaw = newValue.rawValue }
     }
+
+    func update(fromApiModel item: APITab, in client: PersistenceClient, context: Context) throws {
```

Comment:
I think in general we want to name the api -> coredata model function `static func save(_ item: JSON, in context: PersistenceClient) throws -> Self`, but since this adds another parameter `context: Context` which isn't on the api struct, we can't just make this a `WriteableModel`. :(

Reply:
😢